requirements.txt

```diff
@@ -23,7 +23,7 @@ s3fs==0.4.2
 scanpy
 scipy==1.7.3
 tenacity
-tiledb==0.13.1  # Portal's tiledb version should always be the same or older than Explorer's
+tiledb==0.13.2  # Portal's tiledb version should always be the same or older than Explorer's
```

Comment:
do we need to bump explorer as well?

Reply:
Gosh, YES!  I even CR approved that comment! 🤦  https://github.com/chanzuckerberg/single-cell-explorer/pull/242